domains/auth/Auth.API/API/Models/Errors/Errors.cs

```diff
@@ -0,0 +1,14 @@
+namespace API.Errors;
+
+public record AuthError(string ErrorCode, string ErrorDescription)
+{
```

Comment:
Some of these errors arent used. Should they be removed?

Reply:
The errors are taken from the old auth service. 
I find it would be easier to make them now then having to get back to it again.

Reply:
After a small talk, these usued will be deleted